base_location/demo/better_zip.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="demo_brussels" model="res.better.zip">
+      <field name="city">Brussels</field>
```

Comment:
Using it for City management rather than ZIP codes. I don't know any Belgian ZIP codes but I can add one.